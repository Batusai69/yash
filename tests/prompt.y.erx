# printf 'posix PS1={%s}\n' "$PS1"
# printf 'posix PS2={%s}\n' "$PS2"
# printf 'posix PS4={%s}\n' "$PS4"
# exit
# printf 'yash PS1={%s}\n' "$PS1"
# printf 'yash PS2={%s}\n' "$PS2"
# printf 'yash PS4={%s}\n' "$PS4"
# exit
