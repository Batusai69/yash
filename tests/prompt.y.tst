# prompt.y.tst: test of interactive prompt
# vim: set ft=sh ts=8 sts=4 sw=4 noet:

$INVOKE $TESTEE -iv +m -o posix <<\END
printf 'posix PS1={%s}\n' "$PS1"
printf 'posix PS2={%s}\n' "$PS2"
printf 'posix PS4={%s}\n' "$PS4"
exit
END
$INVOKE $TESTEE -iv +m --norcfile <<\END
printf 'yash PS1={%s}\n' "$PS1"
printf 'yash PS2={%s}\n' "$PS2"
printf 'yash PS4={%s}\n' "$PS4"
exit
END
