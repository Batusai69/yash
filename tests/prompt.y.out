posix PS1={$ }
posix PS2={> }
posix PS4={+ }
yash PS1={\$ }
yash PS2={> }
yash PS4={+ }
